scalableminds/webknossos
# Find data improvement

- find data now also returns the resolution in which found the data
- the front end automatically zooms to the resolution in which the backend found data

### URL of deployed dev instance (used for testing):
- https://finddataroute.webknossos.xyz

### Steps to test:
- move to a position where there is no data and then use the "find my data" button
- if you use a dataset where mag1 is missing, it should zoom to another mag (I tested it locally, probably no need to test this explicitly again, as the overhead to set this up is a bit high)

### Issues:
- fixes #3836 

------
